discord/ext/commands/converter.py

```diff
@@ -301,7 +301,10 @@ class ColourConverter(Converter):
             method = getattr(discord.Colour, arg.replace(' ', '_'), None)
             if method is None or not inspect.ismethod(method):
                 raise BadArgument('Colour "{}" is invalid.'.format(arg))
-            return method()
+            try:
+                return method()
+            except ValueError:
```

Comment:
```suggestion
            except TypeError:
```
I said TypeError is raised here

Reply:
I think it may actually be possible to also cause a ValueError, may need some testing. For now I'll just replace it